code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -18,6 +18,7 @@ Mineral Sheets
 /obj/item/stack/sheet/mineral
 	w_type = RECYK_METAL
 	var/recyck_mat
+	perunit = 2000 //See setup.dm, CC_PER_SHEET_MISC

```

Comment:
Are you fucking kidding me?

FUCKING USE THE DEFINE IF YOU'RE TELLING PEOPLE TO LOOK FOR IT.